src/Skeletor/Packages/PixelfusionGitHooksPackage.php

```diff
@@ -12,7 +12,21 @@ public function setup()
         $this->setPackageOptions('--dev');
     }
 
-    public function configure(Framework $activeFramework)
+    public function preInstall()
     {
+        $this->runManager->runCommand('git init');
```

Comment:
Since we are initialising git here, the class name should just be PixelfusionGitPackage, drop Hooks. So here we do everything related with git.

Reply:
@titopixelfusion The class name is based on the name of the package (i.e. `pixelfusion/git-hooks`). In this case I had to init git or else it can't copy the git hooks. Wasn't too happy with it either but couldn't think of a way around it.

Reply:
Makes sense @kielabokkie . If that's the case, good idea to name after the package.